packages/discord.js/typings/index.d.ts

```diff
@@ -5040,7 +5040,12 @@ export interface EmbedField {
   inline: boolean;
 }
 
-export type EmojiIdentifierResolvable = string | EmojiResolvable;
+export type EmojiIdentifierResolvable =
+  | EmojiResolvable
+  | `${string}:${Snowflake}`
+  | `<:${string}:${Snowflake}>`
+  | `<a:${string}:${Snowflake}>`
```

Comment:
You can simplify the two into a single one:
```suggestion
  | `<${'' | 'a'}:${string}:${Snowflake}>`
```

That being said, this is all union'd with `string`, so the string template types are... ignored.